docs/API.md

```diff
@@ -381,11 +381,12 @@ Each *test* has a unique *test id*.
 
 Basic data type: object
 
-The object has three keys, `"module"`, `"message"` and `"level"`.
+The object has four keys, `"module"`, `"message"`, `"level"` and `"testcase"`.
 
 * `"module"`: a string. The *test module* that produced the result.
 * `"message"`: a string. A human-readable *message* describing that particular result.
 * `"level"`: a [*severity level*][Severity level]. The severity of the message.
+* `"testcase"`: a string. The *test case ID* of the *test case* that produced the result.
```

Comment:
I missed this. The terms should be consistent.
```
* `"testcase"`: a string. The *[Test Case Identifiers]* of the *[Test Case][Test Cases]* that produced the result.
```

Reply:
Done